tools/README.md

```diff
@@ -1,16 +1,12 @@
-Hello,
-
-please update this file if you want to upgrade the binaries.
-[Johannes Hoppe](https://github.com/johanneshoppe) will double check the content of the submitted files with the content of the download.
-PRs without this proof will be rejected for security reasons!
-
-See:
-* https://www.mongodb.org/dl/linux/x86_64
-* https://www.mongodb.org/dl/osx
-* https://www.mongodb.org/dl/win32/x86_64-2008plus-ssl
-
-## Used binaries (mongod, mongoexport, mongoimport)
-
-* [mongodb-linux-x86_64-4.0.2](http://downloads.mongodb.org/linux/mongodb-linux-x86_64-4.0.2.tgz?_ga=2.226738654.2109637469.1528191926-586158846.1509018881) (sha256: b1088b3fc0f0f7d4dfe791625175bcd3bbf67209edbed25233d3f67733c5e2ed)
-* [mongodb-osx-ssl-x86_64-4.0.2](http://downloads.mongodb.org/osx/mongodb-osx-ssl-x86_64-4.0.2.tgz?_ga=2.267591757.2109637469.1528191926-586158846.1509018881) (sha256: 737ec3399982c21f6a3722821959d0bd7c5fd046172303fc19551abbd21f126b)
-* [mongodb-win32-x86_64-2008plus-ssl-4.0.2](http://downloads.mongodb.org/win32/mongodb-win32-x86_64-2008plus-ssl-4.0.2.zip?_ga=2.263446735.2109637469.1528191926-586158846.1509018881) (sha256: 7f69b0a42b668b196705027457e463ddf4ccb17589695187b80dc1f73db8fc36)
\ No newline at end of file
+The binaries in this directory are automatically downloaded with the `MongoDownloader` tool.
+
+In order to download the latest binary:
+
+1. Manually delete the three directories (linux + mac + windows) of the previous mongodb version (in this `tools` directory)
+2. Go into the `Mongo2Go/src/MongoDownloader` directory
+3. Run the downloader with `dotnet run`
```

Comment:
Well, step 2 and 3 are just instructions on how to run the downloader tool, so I don't see how running the tool could be part of running the tool. 🤔

For step 1, I personally don't like to automatically delete directories recursively, it always feels a little dangerous. But it could be added either as part of this pull request or even later if you think it's really worth it.

Reply:
Okay, first I wanted to know your opinion! 🙂